### PR TITLE
Sign up flow

### LINKED ISSUE #4134: Sign Up Flow
## Context
- https://github.com/Joystream/atlas/issues/4132

## Design
 [Link to Figma > ](https://www.figma.com/file/C3hvYPVkZ7TcjAsXfqizSD/User-accounts?type=design&node-id=2076%3A106313&t=0RIY9VRH4z4RRJun-1)

## Scope
- [ ] Email and T&Cs
- [ ] Password
- [ ] Public Profile (update of copy on membership screen)
- [ ] [Conf Screen](https://www.figma.com/file/C3hvYPVkZ7TcjAsXfqizSD/User-accounts?node-id=2100%3A58311&t=oBUYdL7AMN4eijwo-1)
- [ ] Errors
- [ ] Form Validation

### PR DESCRIPTION
**Pull request by drillprop**
Fix #4134

### CODE REVIEW

File: packages/atlas/src/components/_overlays/Modal/Modal.styles.ts [hunk 1]
```diff
@@ -33,7 +33,8 @@ export const ModalContent = styled.div<ModalContentProps>`
   position: fixed;
   width: 100vw;
   max-height: calc(100vh - var(--size-topbar-height));
-  overflow: hidden;
+
+  /* overflow: hidden; */
```
Comment (WRadoslaw) [inline packages/atlas/src/components/_overlays/Modal/Modal.styles.ts]: Comment

**Review by WRadoslaw — CHANGES_REQUESTED** [review]
1. Should we show password on input focus when it's hidden?

image

2. In designs, password screen don't have scrollbar and divider

image


3. Not sure why I cannot create a membership (request to the faucet is not triggered)

image

**Comment by drillprop** [comment]
> 2. In designs, password screen don't have scrollbar and divider

I don't think that's true. https://www.figma.com/file/ymHYOFItrmFaLeBC5Z4tpQ/Sign-up-%2F-Log-in?type=design&node-id=2939-77080&t=AcDsClQRTl3N7XH9-4

Check the instructions for RWD 
image

**Comment by WRadoslaw** [comment]
> > 2. In designs, password screen don't have scrollbar and divider
> 
> I don't think that's true. https://www.figma.com/file/ymHYOFItrmFaLeBC5Z4tpQ/Sign-up-%2F-Log-in?type=design&node-id=2939-77080&t=AcDsClQRTl3N7XH9-4
> 
> Check the instructions for RWD image

If so I'd add some spacing on the bottom, rn it looks kind of awkward.

image

**Comment by drillprop** [comment]
> 3. Not sure why I cannot create a membership (request to the faucet is not triggered)

For some reason, it's happening only on production. Investigating...

**Comment by drillprop** [comment]
> > 3. Not sure why I cannot create a membership (request to the faucet is not triggered)
> 
> For some reason, it's happening only on production. Investigating...
 
Fixed. I also introduced some adjustments in your Login Modal because I spotted some styling inconsistencies

**Review by WRadoslaw — APPROVED** [review]

**Pull request merged**